free-2.md

```diff
@@ -11,41 +11,58 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL for avid sproting parents and the sporting community that allows for quick and better stats keepeing for sporting events. The typical sporting parent who wants to keep track of all their kids stats or even high school coaches that are responsable for keeping track of stats for games currently use a peice of paper, a notes app, excell and varios webapages to help keep track of stats for sporting events. Now imagine a DSL that allowed for parents and coaches to easilly add one stat per player at a time in live action and after the game is over have a collection a each players stats and an accumlitive stat sheet representative of the whole team. Moreover, imagine this DSL could be used for highschool through college level programs that currently manually enter stats into their programs webpage. Now they can use the DSL in real time and automaitically tranfer the data to their programs webpage without any manual labor after the game has ended. 
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A dsl that could be accessable during game time would allow for users to update statistics in real time. The need for real time statisic updates at the non profesional level is fairly high. It would be amazing if such a laguage could be used to provide accurate statistics for players coaches and familly members in real time. As well as get rid of the need to record the statisic and maunally input each stat into some interface later on. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this idea is its ability to start small lets say for the average atheletes dad recording player stats for their children, and also have the ability to grow to the level where college coaches who spend hours after a game inputing stats can do it instantly durring the game. I came up with this idea after trying to watch my brothers football games from out of state. My brother plays football in highschool in colorado, the only way I could watch his game was if they put it online. If the game wasnt online I still wanted a way to see how he did. I know my dad takes stats at every game and provides the stats to the coaches too uplaod to maxpreps which is a high school sports stats collection for all teams in the USA. But I can not call him durring the game and ask what are the team stats, and I have to wait a day for these stats to be upaloded onine. There has to be a better way and maybe a DSL could fix this problem.
+
```

Comment:
Oh this is cool! I think I've answered my previous question - real-time stats is a way to check how a team or athlete is doing if you aren't watching the game.